Frends.Community.SQL.Tests/SQLTaskTest.cs

```diff
@@ -303,4 +312,114 @@ public void FormatDbHeader()
                 SQL.FormatDbHeader("123_hello!!! THIS IS 5aNiTiZ3D_MADNESS", true));
         }
     }
+
+    [TestFixture]
+    public class TestBulkInsert
```

Comment:
You should only have one test class in this file, so either move the tests to the above class or create a new file.

Reply:
The test class contains SetUp and TearDown that shouldn't be executed for the old test cases, so I've separated it into a different file.